source/OutfitterPanel.cpp

```diff
@@ -782,30 +793,37 @@ void OutfitterPanel::ToggleForSale()
 {
 	showForSale = !showForSale;
 
+	if (selectedOutfit && !HasItem(selectedOutfit->Name()))
+	{
+		selectedOutfit = nullptr;
+	}
+
 	ShopPanel::ToggleForSale();
 }
 
 
 
-void OutfitterPanel::ToggleCargo()
+void OutfitterPanel::ToggleStorage()
 {
-	if(playerShip)
-	{
-		previousShip = playerShip;
-		playerShip = nullptr;
-		previousShips = playerShips;
-		playerShips.clear();
-	}
-	else if(previousShip)
+	showStorage = !showStorage;
```

Comment:
The ToggleCargo() function had a side-effect of enabling buying to Cargo by unselecting all ships. I don't see how buying to cargo works after this change (since I don't see how to unselect all ships after this change).

Reply:
I found the way to do this by reading the code; pressing control and then selecting the only selected ship will cause that ship to be "un-selected".